MapboxNavigation/Simulation.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import CoreLocation
+
+protocol SimulationDelegate : class {
+    func simulation(_ locationManager: CLLocationManager, didUpdateLocations locations: [CLLocation])
+}
+
+class Simulation : NSObject {
+    
+    var speed: CLLocationSpeed = 10
+    
+    var distanceFilter: CLLocationDistance = 10
+    
+    var verticalAccuracy: CLLocationAccuracy = 40
+    
+    var horizontalAccuracy: CLLocationAccuracy = 40
```

Comment:
Yes, I think adding some noise both to the coordinates and the accuracy will be great for testing snapping functionality but let's make sure this works properly first.